### ISSUE TITLE
AtomSets | Validation set splitting

### ISSUE THREAD

**Issue by jcwang587** [materialsvirtuallab/maml#530]
Hi! I am currently applying the AtomSets model for a transfer learning task on my dataset. The performance has been impressive so far, but I have a question regarding the splitting of the dataset.

I've been following the tutorial [notebook](https://github.com/materialsvirtuallab/maml/blob/master/notebooks/atomsets/structure_model.ipynb), which uses a structural vector and an MLP for regression in the end. The code shows that the data was split into a training dataset (`structs[:80]`, `ef[:80]`) to fit the model, but I couldn't find any indication of a validation set being created. Was a validation dataset used, and if not, how was the best model saved?

I would greatly appreciate your response!

**Comment by jcwang587** [comment]
There were optional arguments `val_features` and `val_targets` provided for fitting.
